tests/test_latent_gaussian.py

```diff
@@ -0,0 +1,79 @@
+import itertools
+
+import chex
+import jax
+import jax.scipy.stats as stats
+import numpy as np
+from absl.testing import absltest, parameterized
+
+from blackjax.mcmc.latent_gaussian import init_and_kernel
+
+
+class GaussianTest(chex.TestCase):
+    @parameterized.parameters(
+        itertools.product([1234, 5678], [True, False], [True, False])
+    )
+    def test_gaussian(self, seed, use_inverse, mean):
+        n_samples = 500_000
+
+        KEY = jax.random.PRNGKey(42)
+        D = 5
+        np.random.seed(seed)
+        C = np.random.randn(D, D**2)
+        C = C @ C.T
+        prior_mean = np.random.randn(D) if mean else None
```

Comment:
Could you change this bit to use JAX random as well? Then everything is control by a single seed